symphony/bdk/core/activity/command.py

```diff
@@ -81,3 +80,12 @@ def matches(self, context: CommandContext) -> bool:
 
     async def on_activity(self, context: CommandContext):
         await self._callback(context)
+
+    def __eq__(self, o: object) -> bool:
+        if isinstance(o, SlashCommandActivity):
```

Comment:
do we need to check if "o" is None ?

Reply:
No, if `o` is `None`, it is of type `NoneType` and `isinstance(o, SlashCommandActivity)` will return `False`.
Tested on the Python console:
<img width="206" alt="Screenshot 2021-06-01 at 09 32 04" src="https://user-images.githubusercontent.com/66254040/120284136-3bed9c00-c2bc-11eb-8776-63b051b82999.png">
.